lab2_task_2.py

```diff
@@ -0,0 +1,16 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+
+# TODO Рассчитайте подушку безопасности, чтобы протянуть 10 месяцев без долгов
+
+i = 1
+money_capital = 0
+while i <= 10:
```

Comment:
```suggestion
while i <= months:
```
Так как вы ранее ввели переменную обозначающую количество месяцев, то стоит ее и использовать